config/locales/devise.he.yml

```diff
@@ -2,9 +2,9 @@
 he:
   devise:
     confirmations:
-      confirmed: כתובת הדואל אומתה בהצלחה.
-      send_instructions: נשלח אליך דואל עם הוראות לאימות כתובת הדואל שאמור להתקבל בדקות הקרובות.
-      send_paranoid_instructions: אם כתובת הדואל שלך קיימת במסד הנתונים, יתקבל בדקות הקרובות דואל עם הוראות לאימות כתובתך.
+      confirmed: 'כתובת הדוא"ל אומתה בהצלחה.'
```

Comment:
לדעתי זה הלחם ולא ראשי תיבות. אישית אני נגד.
In Engish I also write email or eMail, never e-mail.